roles/setup_acm_agents/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+setup_acm_agents_location_label: ""
+setup_acm_agents_inject_dns: false
+setup_acm_agents_no_log: false
+setup_acm_inject_dns: false
```

Comment:
This variable seems to be a duplicate of "setup_acm_agents_inject_dns" with a misspelled name.

Since this is the only plance where this variable is referenced in the PR, I guess it was added by mistake.